tests/Endpoints/SessionTypeTest.php

```diff
@@ -125,24 +124,16 @@ public function graphQLFiltersToTest(): array
         return $filters;
     }
 
-    public function removingSessionThrowsError(array $data)
```

Comment:
this was dead/orphaned code - this method is never executed. it's not a test, since the method name doesn't start with `test`. i tried to turn it into one, but it broke.
so i removed it altogether. 
iirc, there was another few of those.

```
stefan@nichtsnutz: ~/dev/projects/ilios on master[$]
$ grep -r 'removingSessionThrowsError' tests/
tests/Endpoints/SessionTypeTest.php:    public function removingSessionThrowsError(array $data)
```

Reply:
not a test either. 

'''
stefan@nichtsnutz: ~/dev/projects/ilios on master[$]
$ grep -r 'findUsersWithRoleOne' tests/
tests/Endpoints/UserTest.php:    public function findUsersWithRoleOne()
tests/Endpoints/UserTest.php:    public function findUsersWithRoleOneAndFour()
```

Reply:
Ah, nice. Checks out!